source/elements/oneMKL/source/domains/dft/compute_backward.rst

```diff
@@ -317,17 +321,19 @@ compute_backward (USM version)
 
    inout_re
       Pointer to USM allocation of sufficient capacity to store the elements
-      defining the real parts of all the relevant data sequences, as
-      configured by ``desc``. ``data_type`` must be single or double precision floating-point, as 
-      described by the descriptor's precision. Only with complex descriptors configured for in-place
+      defining the real parts of all the relevant data sequences, as configured
+      by ``desc``. ``data_type`` must be single or double precision
+      floating-point, as described by the descriptor's precision. Only with
+      complex descriptors configured for in-place
       operations with :ref:`onemkl_dft_complex_storage_real_real`.
```

Comment:
There are just a couple of lines that could be formatted here.

Reply:
Done in  [554d0fa](https://github.com/uxlfoundation/oneAPI-spec/pull/544/commits/554d0fad2c54c181ade0f8aebbdc30c06c39b1dd).